nas_spec/components/schemas/Payments/3dsEnrollmentData.yaml

```
@@ -12,3 +12,7 @@ properties:
         * `N` - Customer not enrolled
         * `U` - Unknown
     example: Y
+  upgrade_reason:
+    type: string
+    description: Indicates the reason why the payment was upgraded to a 3D secure payment (when `allow_upgrade` is specified)
```

Wording can be simplified a tiny bit
```suggestion
    description: Indicates the reason why the payment was upgraded to 3D secure (when `allow_upgrade` is specified)
```